osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -20,6 +24,8 @@ public class PlaySongSelect : SongSelect
         private OsuScreen player;
         private readonly ModSelectOverlay modSelect;
         private readonly BeatmapDetailArea beatmapDetails;
+        private IEnumerable<Mod> originalMods;

```

Comment:
The behavior in stable: when Auto selected manually and returned from player, the Auto is deselected too.
I think this behavior is troubling in some occasions (like mouse-only presentation), so this marking how the Auto mod is selected.